run.py

```diff
@@ -92,10 +107,14 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS, model_flops=None, num_iter=10):
 
     # if model_flops is not None, output the TFLOPs per sec
     if model_flops:
-        flops, batch_size = model_flops
-        tflops = flops * batch_size / (cpu_walltime / 1.0e9) / 1.0e12
+        if dcgm_enabled:
+            tflops = np.median(flops_records)
```

Comment:
Are we still using median flops here, or are we measuring all 10 iterations and using the average?